freqtrade/rpc/telegram.py

```diff
@@ -750,17 +750,18 @@ def _locks(self, update: Update, context: CallbackContext) -> None:
         Handler for /locks.
         Returns the currently active locks
         """
-        locks = self._rpc._rpc_locks()
-        message = tabulate([[
-            lock['id'],
-            lock['pair'],
-            lock['lock_end_time'],
-            lock['reason']] for lock in locks['locks']],
-            headers=['ID', 'Pair', 'Until', 'Reason'],
-            tablefmt='simple')
-        message = f"<pre>{escape(message)}</pre>"
-        logger.debug(message)
-        self._send_msg(message, parse_mode=ParseMode.HTML)
+        rpc_locks = self._rpc._rpc_locks()
+        for locks in chunks(rpc_locks['locks'], 25):
```

Comment:
what's the 25 based on? random number or experimentation?

I don't mind having this in particular - but i'm asking because telegram has a pretty well-defined message length (4096) (which obviously applys AFTER the formatting) - so i'm not sure if an arbitrary 25 lines will work reliably...

Reply:
My goal was to prouce more readable (as in: not tooo long) message instead of tracking the largest possible message size.
Maybe a mix of limits would be the best idea? Splitting tabulates into messages that are smaller than the max message size but also have a maximum number of lines. Maybe it would even make sense to have this configurable? I'm using telegram on my phone, other people might use a client on their computer with a much bigger screen. Wouldn't mind to implement that.

Reply:
i'm not sure if a configuration will help much.

Splitting it after 25 lines will be fine if you have longer output - it'll only be slightly strange if you have 26 or 27 lines - so the 2nd message is rather short i guess.

if you have it configurable however you'll not gain much (other than configuration complexity) - as i can set it to 26 - but then i have 27 lines - or i configure it to 27, and get 28 lines :laughing: 

---

I guess the only other messages that can be impacted / relevant for this are `/trades` and `/edge` (although this is slightly different as it tabulates a dict, not a list).
for `/performance` the splitting is already done - but based on "max-messagelength" - not per line.